### ISSUE TITLE
Anima cost X gold value of equipment or augment runes

### ISSUE THREAD

**Issue by theruan** [TLDRMissions/TLDRMissions#621]
TLDR version: 10.2-004

Ideally running the mission to sell the equipment with a more detailed anima/gold would be better.

In the current system if a mission costs 53(total) and would sell for 99g i could select to not do it, but it isn't a bad choice(at least not for me), i don't select the range cause in some cases missions like 2204 can cost 50+ anima and end up giving a 50g item, a 50g would be ok if the mission was to cost 20~30 anima.
For this reason a slider(maybe named as "npc ratio value") with a ratio, not only for equipment, but now also for augment runes(considering they may end up only being npc trash too.

Lets say, mission X costs 33 anima total and gives 25g and 5 aug runes, so 25g + 5 * 11,2752g = 25 + 56,376g = 81,376g and G / Anima = 81,376 / 33 = 2,465 ratio
A slider starting with 0,5(not sure if there is anything around that, but ok) up to 2.5+ should do.
